rebuild.sh

```diff
@@ -172,7 +338,7 @@ mvnBuildLocal() {
 rebuildToolMvn() {
   # the effective rebuild command, adding artifact:buildinfo goal to compare with central content
   #local mvn_rebuild="${command} -V -e artifact:compare -Dbuildinfo.reproducible"
-  local mvn_rebuild="${command} -V -e org.apache.maven.plugins:maven-artifact-plugin:3.3.0:compare -Dbuildinfo.reproducible"
```

Comment:
I'd prefer have this change in a separate PR

Reply:
Ok